### DISCUSSION TITLE
adding boolean as custom field

### DISCUSSION THREAD

**Discussion by dylm64** [avitorio/outstatic#233] [Ideas]
I needed a boolean field to have a list of highlight posts.
I made it in my fork, now checking if it could be useful to add it and create a pr for it in the original repo.

**Comment by avitorio** [comment]
Hey, that would be helpful! Feel free to open a PR!

**Reply by avitorio** [reply]
@dylm64 this is now live on the latest release `v1.4.9` - Thank you!
